mmpose/structures/utils.py

```diff
@@ -120,6 +120,9 @@ def split_instances(instances: InstanceData) -> List[InstanceData]:
     information about one instance."""
     results = []
 
+    if instances is None:
```

Comment:
Please add a few comments to explain this is the case where no object is detected.